src/main/java/justin/command/AddDeadlineCommand.java

```diff
@@ -0,0 +1,68 @@
+package justin.command;
+
+import justin.task.Deadline;
+import justin.DukeException;
+import justin.Storage;
+import justin.task.Task;
+import justin.TaskList;
+import justin.Ui;
+
+/**
+ * Represents a command that is called to add a deadline
+ * task in the TaskList, and save changes.
+ * @author Justin Cheng.
+ */
+public class AddDeadlineCommand extends Command {
+    private String description;
+    private boolean isDone;
+    private String by;
```

Comment:
Perhaps the naming for the `by` variable can be more consistent with that of the `time` variable? I notice that both refer to time (technically one date and one time), so maybe for readability they should have similar names, because "by" doesn't really seem to relate to "time".

Reply:
Good naming to show variable is a boolean.